zeppelin-web/src/components/searchService/search.service.js

```diff
@@ -26,4 +26,6 @@ angular.module('zeppelinWebApp').service('searchService', function($resource, ba
     });
   };
 
+  this.searchTerm = '';
```

Comment:
You will need to do a `this.searchTerm = term.q;` at the beginning of the `this.search` function, otherwise there is no search field when you reload the page